New package: AnyLogic.AnyLogic.Professional version 8.7.12

mdanish-kh opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/66123)

Commit:
New package: AnyLogic.AnyLogic.Pro version 8.7.12

Trenly commented on manifests/a/AnyLogic/AnyLogic/Pro/8.7.12/AnyLogic.AnyLogic.Pro.locale.en-US.yaml:
```diff
@@ -0,0 +1,34 @@
+# CopyrightUrl: 
+ShortDescription: AnyLogic is a multimethod, java-based simulation modeling tool developed by The AnyLogic Company.
+Description:  AnyLogic is a multimethod, java-based simulation modeling tool developed by The AnyLogic Company. It supports agent-based, discrete event, and system dynamics simulation methodologies.
+Moniker: anylogic
```
```suggestion
Moniker: anylogic-professional
```

Trenly commented on manifests/a/AnyLogic/AnyLogic/Pro/8.7.12/AnyLogic.AnyLogic.Pro.installer.yaml:
```diff
@@ -0,0 +1,22 @@
+- Architecture: x64
+  InstallerUrl: https://files.anylogic.com/anylogic-professional-8.7.12.x86_64.exe
+  InstallerSha256: 2C191295BB4DC8FB93D175CD7E990BBEF8FF90DC6D1B6AF68D8343E505005EE0
+- Architecture: x86
+  InstallerUrl: https://files.anylogic.com/anylogic-professional-8.7.12.x86_64.exe
+  InstallerSha256: 2C191295BB4DC8FB93D175CD7E990BBEF8FF90DC6D1B6AF68D8343E505005EE0
```
```suggestion
```

Trenly commented on manifests/a/AnyLogic/AnyLogic/Pro/8.7.12/AnyLogic.AnyLogic.Pro.installer.yaml:
```diff
@@ -0,0 +1,22 @@
+FileExtensions:
+- alp
+- java
+ReleaseDate: 2022-05-25
```
```suggestion
ReleaseDate: 2022-05-25
AppsAndFeaturesEntries: 
- DisplayName: AnyLogic 8.7.12 Professional
```

Commit:
Fix AnyLogic.AnyLogic.Professional version 8.7.12

- Change package name 'Pro' to 'Professional'
- Remove x86 installer entry
- Add AppsAndFeaturesEntries
- Rename directory and manifest files
- Update moniker 'anylogic' to 'anylogic-professional'
